src/__test__/utils/helpers.spec.js

```diff
@@ -0,0 +1,587 @@
+    const countMock = 2
+
+    describe('Default call', () => {
+      test('Should return a counter with the pluralized noun', () => {
+        const result = pluralize(countMock, nounMock)
+        expect(result).toBe(`${countMock} ${nounMock}s`)
+      })
+    })
+
+    describe('"stripCount" param', () => {
+      test('Should return the pluralized noun without the counter if the "stripCount" flag is "true"', () => {
+        const stripCountMock = true
+        const result = pluralize(countMock, nounMock, stripCountMock)
+        expect(result).toBe(`${nounMock}s`)
+      })
+
+      test('Should return the pluralized noun with the counter if the "stripCount" flag is "false"', () => {
+        const stripCountMock = false
+        const result = pluralize(countMock, nounMock, stripCountMock)
+        expect(result).toBe(`${countMock} ${nounMock}s`)
+      })
+    })
+
+    describe('"suffix" param', () => {
+      test('Should return a counter with the pluralized noun using a custom "suffix"', () => {
+        const stripCountMock = false
+        const suffixMock = 'random'
+        const result = pluralize(countMock, nounMock, stripCountMock, suffixMock)
+        expect(result).toBe(`${countMock} ${nounMock}${suffixMock}`)
+      })
+    })
+  })
+
+  describe('- Counter is equal to 1', () => {
+    const countMock = 1
+
+    describe('Default call', () => {
+      test('Should return a counter with the plain noun', () => {
+        const result = pluralize(countMock, nounMock)
+        expect(result).toBe(`${countMock} ${nounMock}`)
+      })
+    })
+
+    describe('"stripCount" param', () => {
+      test('Should return the plain noun without the counter if the "stripCount" flag is "true"', () => {
+        const stripCountMock = true
+        const result = pluralize(countMock, nounMock, stripCountMock)
+        expect(result).toBe(`${nounMock}`)
+      })
+    })
+  })
+})
+
+describe('#size()', () => {
+  test('Should be an instance of Function', () => {
+    expect(size).toBeInstanceOf(Function)
+  })
+
+  describe('- "collection" param', () => {
+    test('Should return "0" if "collection" param is not passed', () => {
+      const result = size()
+      expect(result).toBe(0)
+    })
+
+    test('Should return "0" if "collection" param is not passed', () => {
+      const result = size()
+      expect(result).toBe(0)
+    })
+
+    test('Should return the length for a "collection" param of "String" type', () => {
+      const collection = 'randomCollectionStringWord'
+      const result = size(collection)
+      expect(result).toBe(collection.length)
+    })
+
+    test('Should return the length for a "collection" param of "Array" type', () => {
+      const collection = ['a', 'b', 'c', 'd', 'e', 1, 2, 3, 4, 5]
+      const result = size(collection)
+      expect(result).toBe(collection.length)
+    })
+
+    test('Should return the length for a "collection" param of "Object" type', () => {
```

Comment:
Please note that existing tests use lower case for `test` statements. `should` is redundant here as the test already reads great: `describe('smth').test('return smth when smth else')`
```suggestion
    test('return the length for a "collection" param of "Object" type', () => {
```

Reply:
done